custom-connectors/Tikit/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "ApiKey",
+          "description": "The ApiKey for this api",
+          "tooltip": "Provide your ApiKey",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#9ab5dd",
+    "capabilities": [],
+    "publisher": "Celebaltech"
```

Comment:
@1MrSharma Please add `"stackOwner"` property. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

Reply:
Added stackOwner property.